Utils/Services/ImageFetcher.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  ImageFetcher.swift
+//  Utils
+//
+//  Created by Nahuel Gladstein on 6/13/17.
+//  Copyright © 2017 Wolox. All rights reserved.
+//
+
+import UIKit
+import ReactiveSwift
+
+public protocol ImageFetcherType {
+    
+    func fetchImage(_ imageURL: URL) -> SignalProducer<UIImage, ImageFetcherError>
+    
+}
+
+public enum ImageFetcherError: Error {
+    case invalidImageFormat
+    case fetchError(Error)
+}
+
+public class ImageFetcher: ImageFetcherType {
+    
+    /**
+     Fetches an image asynchronously from a URL, returning a value or an error in the SignalProducer when done.
+     - parameter imageURL: the url of the image to fetch.
+     */
+    public func fetchImage(_ imageURL: URL) -> SignalProducer<UIImage, ImageFetcherError> {
+        return URLSession.shared
```

Comment:
With the old swift and `NSURLSession` you could do something like:
```
let configuration = NSURLSessionConfiguration.defaultSessionConfiguration()
configuration.requestCachePolicy = .ReturnCacheDataElseLoad
let session = NSURLSession(configuration: configuration)
```
so that images are cached. But maybe it's better to use the shared one (saying so in the documentation) so that if they want to cache the images, they should make the shared one cache (I think the shared one can be changed, right?)